ChatTTS/core.py

```diff
@@ -519,65 +527,57 @@ def _infer_code(
             )
             input_ids = [i.tolist() for i in input_ids]
 
-            result = gpt.llm.generate(
-                None,
-                sample_params,
-                input_ids,
+            results_generator = gpt.llm.llm_engine.generate(
```

Comment:
It's not so good to call into the inner method and you should modify the outer one `gpt.llm.generate`.